src/lib/commands/config.ts

```diff
@@ -23,7 +22,12 @@ const showRenameAccountPrompt = (accountToRename: Portal) => {
     .then(async (newName: string | undefined) => {
       if (newName) {
         const oldName = accountToRename.name || accountToRename.portalId;
-        const invalidReason = portalNameInvalid(newName, getConfig());
+        const config = getConfig();
+        let invalidReason = '';
+        if (config) {
+          // @ts-expect-error TODO: Fix this when updating local-dev-lib
```

Comment:
The deprecated config and new config types are simply incompatible with one another. We can remove this error, once we account for the new global config in the extension.